tests/guessTheNumber_test.js

```diff
@@ -13,4 +13,12 @@ describe('guessTheNumber()', () => {
     const randomNum = Math.floor(Math.random() * 100)
     expect(guessTheNumber('')).to.equal('Insert only integers RAWR!!!')
   })
+  it('returns string if input < randomNum', () => {
```

Comment:
This seems like it belongs in a different branch and PR (guessTheNumber). It is best practice to keep the changes in a branch related to its name.